src/client/dom-processor/client-dom-adapter.js

```diff
@@ -99,7 +99,7 @@ export default class ClientDomAdapter extends BaseDomAdapter {
     }
 
     isTopParentIframe (el) {
-        var elWindow = el[DOM_SANDBOX_PROCESSED_CONTEXT];
+        var elWindow = el[INTERNAL_ATTRS.processedContext];
```

Comment:
Ok, as I can see, here we have property, not attribute. So, seems, like we need to split constants in two groups: internal properties and internal attributes. We can put them in `processing/dom/internal/properties.js' and `processing/dom/internal/attributes.js' respectively.